GenderPayGap.WebUI/Views/AdminOrganisationSicCodes/ConfirmSicCodeChanges.cshtml

```diff
@@ -92,45 +89,59 @@
         else
         {
             <table class="govuk-table">
+                <thead class="govuk-table__head govuk-visually-hidden">
+                    <tr class="govuk-table__row">
+                        <th scope="col" class="govuk-table__header govuk-!-padding-0">
+                            <span class=" govuk-visually-hidden">SIC codes to add</span>
+                        </th>
+                    </tr>
+                </thead>
                 <tbody class="govuk-table__body">
                     @foreach (SicCode sicCode in Model.SicCodesToAdd)
                     {
                         <tr class="govuk-table__row">
                             <td class="govuk-table__cell">
                                 <b>@sicCode.SicCodeId</b>
                                 @sicCode.Description
-                                <br />
+                                <br/>
                                 <span class="govuk-body-s">
                                     part of: @sicCode.SicSection.Description
                                 </span>
-                                <input type="hidden" name="@nameof(ChangeOrganisationSicCodesViewModel.SicCodeIdsToAdd)" value="@sicCode.SicCodeId" />
+                                <input type="hidden" name="@nameof(ChangeOrganisationSicCodesViewModel.SicCodeIdsToAdd)" value="@sicCode.SicCodeId"/>
                             </td>
                         </tr>
                     }
                 </tbody>
             </table>
         }
 
-
-        <h3 class="govuk-heading-m">SIC codes to keep</h3>
-
+        <h3 class="govuk-heading-m govuk-!-margin-bottom-2">
+            SIC codes to keep
+        </h3>
         @if (Model.SicCodesToKeep.Count == 0)
         {
             <p class="govuk-body">
-                No SIC codes to keep
+                No SIC codes to add
             </p>
         }
         else
         {
             <table class="govuk-table">
+                <thead class="govuk-table__head">
+                    <tr class="govuk-table__row">
+                        <th scope="col" class="govuk-table__header govuk-!-padding-0">
+                            <span class=" govuk-visually-hidden">SIC codes to keep</span>
+                        </th>
+                    </tr>
+                </thead>

```

Comment:
Would it be worth moving the `govuk-visually-hidden` from the `span` to the `thead` (and maybe removing the `span` altogether (you could put the text directly inside the `th`))?
If you think it's better here, then happy to discuss